GitHub Security Advisory ID (GHSA-w3x6-4m5h-cxqf) reported on branch release-10.0

A security vulnerability, [GHSA-w3x6-4m5h-cxqf](https://github.com/advisories/GHSA-w3x6-4m5h-cxqf), has been reported on a dependency used by a project in the `release-10.0` branch of this repository.

> [!NOTE]
> This does not necessarily mean any released packages or applications from this branch use the dependency in a way that exposes the vulnerability. 

It is recommended for users to investigate and take action if necessary until the dependency can be updated.

A PR to update the dependency to a non-vulnerable version will be raised as soon as possible. Not all dependencies in all branches require a release, but if one is required it will be published shortly after the PR is merged.

Subscribe to this issue to get updates on Particular's work to address this dependency vulnerability.

## Symptoms

When a project has the setting [NuGetAuditMode](https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages#configuring-nuget-audit) set to `all`, at build time there are warnings about vulnerable transitive dependencies related to packages released from this repository.

## How to take immediate action if necessary

Microsoft has detailed [guidance on how to take action](https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages#actions-when-packages-with-known-vulnerabilities-are-reported) when a known vulnerability is reported.

<!-- GHSA-w3x6-4m5h-cxqf / release-10.0 -->

Could this please be picked up? Almost all applications at my company use this package and our pipelines are currently failing on the dotnet restore step due to this issue. It should just be a simple package update.

`error NU1903: Warning As Error: Package 'System.Security.Cryptography.Xml' 10.0.0 has a known high severity vulnerability`

Thanks in advance!

--- a/src/NServiceBus.Testing/NServiceBus.Testing.csproj
+++ b/src/NServiceBus.Testing/NServiceBus.Testing.csproj
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="10.0.0" />
+    <PackageReference Include="NServiceBus" Version="10.1.4" />
     <PackageReference Include="Particular.Packaging" Version="4.5.0" PrivateAssets="All" />
   </ItemGroup>
 

